app/helpers/template_helpers.py

```diff
@@ -14,25 +14,144 @@
+                ),
+            },
+            "rows": [
+                {"itemsList": self.footer_items},
+            ],
+            "legal": [
+                {
+                    "itemsList": [
+                        {
+                            "text": lazy_gettext("Cookies"),
+                            "url": self.static_content_urls["cookies"],
+                            "target": "_blank",
+                        },
+                        {
+                            "text": lazy_gettext("Accessibility statement"),
+                            "url": self.static_content_urls["accessibility_statement"],
+                            "target": "_blank",
+                        },
+                        {
+                            "text": lazy_gettext("Privacy and data protection"),
+                            "url": self.static_content_urls[
+                                "privacy_and_data_protection"
+                            ],
+                            "target": "_blank",
+                        },
+                        {
+                            "text": lazy_gettext("Terms and conditions"),
+                            "url": self.static_content_urls["terms_and_conditions"],
+                            "target": "_blank",
+                        },
+                    ],
+                },
+            ],
+        }
+
+        if request.blueprint == "post_submission":
+            context["footerWarning"] = lazy_gettext(
+                "Make sure you <a href='{sign_out_url}'>leave this page</a> or close your browser if using a shared device"
+            ).format(sign_out_url=sign_out_url)
+
+        return context
+
+    @cached_property
+    def footer_items(self):
+        return [
+            {
+                "text": lazy_gettext("Help"),
+                "url": self.static_content_urls["help"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("Contact us"),
+                "url": self.static_content_urls["contact_us"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("Languages"),
+                "url": self.static_content_urls["languages"],
+                "target": "_blank",
+            },
+            {
+                "text": lazy_gettext("BSL and audio videos"),
+                "url": self.static_content_urls["bsl_and_audio_videos"],
+                "target": "_blank",
+            },
+        ]
+
+
+class CensusNISRAContextHelper(ContextHelper):
+    def __init__(self, language: str, base_url: str):
+        super().__init__(language, base_url)
+        for entry in ("languages", "bsl_and_audio_videos"):
+            self.static_content_urls.pop(entry)
+        self.static_content_urls[
+            "help"
+        ] = f"{base_url}help/help-with-the-questions/online-questions-help/"
```

Comment:
I think this is wrong. If `languages` and `bsl_and_audio_videos` don't apply to all contexts then they shouldn't be set in the common class.

Reply:
Done